tests/phpunit/testcases/tests_things/test_thing_glossary.php

```diff
@@ -39,12 +39,129 @@ function test_by_set_or_parent_project() {
 		$this->assertEquals( $glossary, $subsub_glossary );
 		$this->assertEquals( $glossary->path(), $subsub_glossary->path() );
 	}
-	
+
 	function test_delete() {
 		$glossary = GP::$glossary->create_and_select( array( 'translation_set_id' => '1' ) );
 		$glossary->delete();
 		$new = GP::$glossary->by_set_id( '1' );
 		$this->assertNotEquals( $glossary, $new );
-		
+
+	}
+
+	/**
+	 * @ticket gh-435
+	 */
+	function test_get_entries() {
+
+		$this->assertFalse( GP::$glossary->get_entries() );
+
+		$set = $this->factory->translation_set->create_with_project_and_locale();
+		$glossary = GP::$glossary->create_and_select( array( 'translation_set_id' => $set->id ) );
+
+		$test_entries = array(
+			array(
+				'term' => 'Term',
+				'translation' => 'Translation',
+				'glossary_id' => $glossary->id,
+			),
+			array(
+				'term' => 'Term 2',
+				'translation' => 'Translation 2',
+				'glossary_id' => $glossary->id,
+			),
+		);
+
+		GP::$glossary_entry->create_and_select( $test_entries[0] );
+		$entries = $glossary->get_entries();
+
+		$this->assertCount( 1, $entries );
+		$this->assertInstanceOf( 'GP_Glossary_Entry', $entries[0] );
+
+		$this->assertEquals( $entries[0]->term, $test_entries[0]['term'] );
+		$this->assertEquals( $entries[0]->translation, $test_entries[0]['translation'] );
+
+		GP::$glossary_entry->create_and_select( $test_entries[1] );
+
+		// Test that caching is working.
+		$this->assertCount( 1, $glossary->get_entries() );
+
+		$glossary->entries = array();
```

Comment:
`entries` is a private property so this doesn't work, see https://travis-ci.org/GlotPress/GlotPress-WP/jobs/165340711#L338-L352-